CarpetLib/src/prolongate_3d_cc_eno_rf2.cc

```diff
@@ -1289,8 +1289,8 @@ void prolongate_3d_cc_eno_rf2(
 
   } else { // use_loopcontrol_in_operators
 
-// Loop over fine region
-#pragma omp parallel
+    // Loop over fine region
+    // #pragma omp parallel
```

Comment:
this should not be committed in this form. Rather the whole pragma should be removed.